docs/Use-Case-Cyclic-Voltammetry.md

```diff
@@ -0,0 +1,20 @@
+Use Case – cyclic voltammetry experiment for three electrode cell
+
+1. Input Initial Potential (V vs. Eocv)
+2. Input first turnover potential (V vs. Ref)
```

Comment:
I am not sufficiently familiar with the voltametry measurement to know what these things are.  That is ok if all the coders are, but not if they are not, if you see what I mean.